libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationViewEventDispatcher.java

```diff
@@ -24,19 +25,39 @@
   private RouteListener routeListener;
   private BottomSheetCallback bottomSheetCallback;
 
-  void setFeedbackListener(@Nullable FeedbackListener feedbackListener) {
+  /**
+   * Initializes the listeners in the dispatcher, as well as the listeners in the {@link MapboxNavigation}
+   *
+   * @param navigationViewOptions that contains all listeners to attach
+   */
+  void initializeListeners(NavigationViewOptions navigationViewOptions, MapboxNavigation navigation) {
+    updateFeedbackListener(navigationViewOptions.feedbackListener());
```

Comment:
Are we updating the listeners? What about using `assign` instead?

BTW, for consistency we should use `Listener` or `Callback` across the base code. Saw that sometimes we name callbacks as `Listener` and sometimes as `Callback`. This is something that we could revisit later.